webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/OrderedScrambles.java

```diff
@@ -0,0 +1,118 @@
+package net.gnehzr.tnoodle.server.webscrambles;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.LinkedHashMap;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.Days;
+
+import net.lingala.zip4j.exception.ZipException;
+import net.lingala.zip4j.io.ZipOutputStream;
+import net.lingala.zip4j.model.ZipParameters;
+
+import com.google.gson.JsonElement;
+
+import com.itextpdf.text.DocumentException;
+
+public class OrderedScrambles {
+    
+    // TODO we are using Joda-Time for dates, via its .jar file
```

Comment:
nit: Could you file a github issue for this and update this comment to link to it?

Reply:
Done